rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/560_ignored_source.yml

```diff
@@ -0,0 +1,186 @@
+---
+setup:
```

Comment:
maybe move this yaml file to logsdb module? Then we don't have to add yet another node feature.